\nonstopmode{}
\documentclass[a4paper]{book}
\usepackage[times,inconsolata,hyper]{Rd}
\usepackage{makeidx}
\usepackage[utf8,latin1]{inputenc}
% \usepackage{graphicx} % @USE GRAPHICX@
\makeindex{}
\begin{document}
\chapter*{}
\begin{center}
{\textbf{\huge Package `leafcutter'}}
\par\bigskip{\large \today}
\end{center}
\begin{description}
\raggedright{}
\item[Type]\AsIs{Package}
\item[Title]\AsIs{Modified Package for Shiny Display of Leafcutter Analysis
Results}
\item[Version]\AsIs{0.2.7.1}
\item[Date]\AsIs{2021-01-29}
\item[Author]\AsIs{David A. Knowles, Yang Li, Jonathan Pritchard}
\item[Maintainer]\AsIs{David A. Knowles }\email{knowles84@gmail.com}\AsIs{}
\item[Description]\AsIs{Alternative Splicing Quantification, Differential Splicing and Splicing QTL Mapping}
\item[License]\AsIs{GPL (>= 3)}
\item[Depends]\AsIs{R (>= 3.0.2)}
\item[Imports]\AsIs{ggplot2, R.utils, gridExtra, reshape2, Hmisc, foreach, dplyr,
magrittr, stringr, optparse, shiny, shinyjs, DT, intervals,
shinycssloaders, gtable}
\item[RoxygenNote]\AsIs{6.0.1}
\item[Suggests]\AsIs{knitr, rmarkdown}
\item[VignetteBuilder]\AsIs{knitr}
\item[NeedsCompilation]\AsIs{no}
\end{description}
\Rdcontents{\R{} topics documented:}
\inputencoding{utf8}
\HeaderA{add\_chr}{Adding "chr" if it's not present}{add.Rul.chr}
%
\begin{Description}\relax
Adding "chr" if it's not present
\end{Description}
%
\begin{Usage}
\begin{verbatim}
add_chr(chrs)
\end{verbatim}
\end{Usage}
%
\begin{Arguments}
\begin{ldescription}
\item[\code{chrs}] Chromosome or cluster names
\end{ldescription}
\end{Arguments}
%
\begin{Value}
Data.frame with cluster ids and genes separated by commas
\end{Value}
\inputencoding{utf8}
\HeaderA{get\_intron\_meta}{Make a data.frame of meta data about the introns}{get.Rul.intron.Rul.meta}
%
\begin{Description}\relax
Make a data.frame of meta data about the introns
\end{Description}
%
\begin{Usage}
\begin{verbatim}
get_intron_meta(introns)
\end{verbatim}
\end{Usage}
%
\begin{Arguments}
\begin{ldescription}
\item[\code{introns}] Names of the introns
\end{ldescription}
\end{Arguments}
%
\begin{Value}
Data.frame with chr, start, end, cluster id and "middle"
\end{Value}
\inputencoding{utf8}
\HeaderA{leaf\_cutter\_effect\_sizes}{Per intron effect sizes}{leaf.Rul.cutter.Rul.effect.Rul.sizes}
%
\begin{Description}\relax
Per intron effect sizes
\end{Description}
%
\begin{Usage}
\begin{verbatim}
leaf_cutter_effect_sizes(results)
\end{verbatim}
\end{Usage}
%
\begin{Arguments}
\begin{ldescription}
\item[\code{results}] From \code{\LinkA{differential\_splicing}{differential.Rul.splicing}}
\end{ldescription}
\end{Arguments}
%
\begin{Value}
data.frame with a row for every tested intron and columns: intron, log effect size, baseline proportions, proportions in the second condition, and resulting deltaPSI
\end{Value}
\inputencoding{utf8}
\HeaderA{logistic}{logistic (sigmoid) function}{logistic}
%
\begin{Description}\relax
logistic (sigmoid) function
\end{Description}
%
\begin{Usage}
\begin{verbatim}
logistic(g)
\end{verbatim}
\end{Usage}
%
\begin{Arguments}
\begin{ldescription}
\item[\code{g}] The log odds
\end{ldescription}
\end{Arguments}
%
\begin{Value}
1/(1+e\textasciicircum{}-g)
\end{Value}
\inputencoding{utf8}
\HeaderA{make\_differential\_splicing\_plot}{Make sashimi-esque plot with ggplot2}{make.Rul.differential.Rul.splicing.Rul.plot}
%
\begin{Description}\relax
Shows only junction reads and can optionally show splicing variation across groups.
\end{Description}
%
\begin{Usage}
\begin{verbatim}
make_differential_splicing_plot(y, x = numeric(nrow(y)) + 1,
  exons_table = NULL, len = 500, length_transform = function(g) log(g +
  1), main_title = NA, snp_pos = NA, summary_func = colMeans,
  legend_title = "Mean count")
\end{verbatim}
\end{Usage}
%
\begin{Arguments}
\begin{ldescription}
\item[\code{y}] [samples] x [introns] matrix of counts

\item[\code{x}] [samples] vector of group membership. Can be numeric, factor or character.

\item[\code{exons\_table}] An optional data frame specifying exons, with columns: chr start end strand gene\_name. For hg19 see data/gencode19\_exons.txt.gz

\item[\code{len}] Number of segments each curve is constructed from, controls smoothness.

\item[\code{length\_transform}] Function controlling how true genomic space is mapped to the plot for improved visability. Use the identity function (i.e. function(g) g

\item[\code{main\_title}] Title

\item[\code{snp\_pos}] An optional list of SNP positions to mark

\item[\code{summary\_func}] Function to summarize counts within groups: usually colMeans or colSums

\item[\code{legend\_title}] Match this to summary\_func
\end{ldescription}
\end{Arguments}
\inputencoding{utf8}
\HeaderA{map\_clusters\_to\_genes}{Work out which gene each cluster belongs to. Note the chromosome names used in the two inputs must match.}{map.Rul.clusters.Rul.to.Rul.genes}
%
\begin{Description}\relax
Work out which gene each cluster belongs to. Note the chromosome names used in the two inputs must match.
\end{Description}
%
\begin{Usage}
\begin{verbatim}
map_clusters_to_genes(intron_meta, exons_table)
\end{verbatim}
\end{Usage}
%
\begin{Arguments}
\begin{ldescription}
\item[\code{intron\_meta}] Data frame describing the introns, usually from get\_intron\_meta

\item[\code{exons\_table}] Table of exons, see e.g. /data/gencode19\_exons.txt.gz
\end{ldescription}
\end{Arguments}
%
\begin{Value}
Data.frame with cluster ids and genes separated by commas
\end{Value}
\inputencoding{utf8}
\HeaderA{multiqq}{Plot multiple qq plot for p-values}{multiqq}
%
\begin{Description}\relax
Plot multiple qq plot for p-values
\end{Description}
%
\begin{Usage}
\begin{verbatim}
multiqq(pvalues)
\end{verbatim}
\end{Usage}
%
\begin{Arguments}
\begin{ldescription}
\item[\code{pvalues}] A named list of numeric vectors of p-values.
\end{ldescription}
\end{Arguments}
%
\begin{Value}
a ggplot
\end{Value}
\printindex{}
\end{document}
